### PR TITLE
Upgrade cobertura plus minor fixes

### PR DESCRIPTION
**Pull request by duncangrant**
This is the simple change that provides code coverage on a per project basis but doesn't do any coverage merging

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1657,8 +1658,8 @@
             <id>Coverage</id>
             <dependencies>
                 <dependency>
-                    <groupId>net.sourceforge.cobertura</groupId>
-                    <artifactId>cobertura</artifactId>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>cobertura-maven-plugin</artifactId>
                     <version>${cobertura.version}</version>
```
Comment (aledsage) [inline pom.xml]: Why not the latest version `2.6` of the plugin: http://search.maven.org/#artifactdetails%7Corg.codehaus.mojo%7Ccobertura-maven-plugin%7C2.6%7Cmaven-plugin

File: pom.xml [hunk 2]
```diff
@@ -113,7 +113,7 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
         <!-- Testing -->
-        <cobertura.version>1.9.4.1</cobertura.version>
+        <cobertura.version>2.5.2</cobertura.version>
```
Comment (aledsage) [inline pom.xml]: Perhaps worth using `cobertura.plugin.version` or some such, as cobertura itself is still on 2.0.3 so this might confuse people?

**Comment by aledsage** [comment]
Changes look good; just a couple of minor questions. I'll try it out later this afternoon.

File: pom.xml (hunk 1 above)
Comment (duncangrant) [inline pom.xml]: Everything comes up zero using 2.6 - not sure why but it seems to be a known bug and the recommendation is to go back to 2.5.2

**Comment by aledsage** [comment]
Looks good; merging.

But still out of interest, why not the latest plugin version of 2.6: http://search.maven.org/#artifactdetails%7Corg.codehaus.mojo%7Ccobertura-maven-plugin%7C2.6%7Cmaven-plugin

**Pull request merged**